corehq/form_processor/tests/test_case_dbaccessor.py

```diff
@@ -0,0 +1,331 @@
+import uuid
+from datetime import datetime
+
+from django.test import TestCase
+
+from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
+from corehq.form_processor.backends.sql.processor import FormProcessorSQL
+from corehq.form_processor.exceptions import AttachmentNotFound, CaseNotFound
+from corehq.form_processor.models import XFormInstanceSQL, CommCareCaseSQL, \
+    CaseTransaction, CommCareCaseIndexSQL, CaseAttachmentSQL, SupplyPointCaseMixin
+from crispy_forms.tests.utils import override_settings
+
+DOMAIN = 'test-case-accessor'
+
+
+@override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+class CaseAccessorTests(TestCase):
+    dependent_apps = []
+
+    def test_get_case_by_id(self):
+        case = _create_case()
+        with self.assertNumQueries(1):
+            case = CaseAccessorSQL.get_case(case.case_id)
+        self.assertIsNotNone(case)
+        self.assertIsInstance(case, CommCareCaseSQL)
+        self.assertEqual(DOMAIN, case.domain)
+        self.assertEqual('user1', case.owner_id)
+
+    def test_get_case_by_id_missing(self):
+        with self.assertRaises(CaseNotFound):
+            CaseAccessorSQL.get_case('missing_case')
+
+    def test_get_cases(self):
+        case1 = _create_case()
+        case2 = _create_case()
+
+        cases = CaseAccessorSQL.get_cases(['missing_case'])
+        self.assertEqual([], cases)
+
+        cases = CaseAccessorSQL.get_cases([case1.case_id])
+        self.assertEqual(1, len(cases))
+        self.assertEqual(case1.case_id, cases[0].case_id)
+
+        cases = CaseAccessorSQL.get_cases([case1.case_id, case2.case_id], ordered=True)
+        self.assertEqual(2, len(cases))
+        self.assertEqual(case1.case_id, cases[0].case_id)
+        self.assertEqual(case2.case_id, cases[1].case_id)
+
+    def test_case_modified_since(self):
+        case = _create_case()
+
+        self.assertFalse(CaseAccessorSQL.case_modified_since(case.case_id, case.server_modified_on))
+
+        self.assertTrue(CaseAccessorSQL.case_modified_since(case.case_id, datetime.utcnow()))
```

Comment:
Should this actually return false? To me this says: "Has the case been modified after `datetime.utcnow()`?", and I think the answer to that question would be "no". I might be misunderstanding the meaning of `case_modified_since()` or something else though...